src/containers/Home.tsx

```diff
@@ -1,13 +1,15 @@
 import { useTranslation } from "react-i18next";
 
 import FeedPost from "../components/FeedPost";
+import TweetCreationForm from "../components/TweetCreationForm";
 
 const HomeContainer: React.FunctionComponent = () => {
   const { t } = useTranslation();
   return (
     <>
       <div>Home container</div>
       <p>{t("test")}</p>
+      <TweetCreationForm />
```

Comment:
Why are you render <TweetCreationForm /> in Home page? It because we have no button for it?

Reply:
it was temporary, deleted it